app/controllers/alchemy/admin/contents_controller.rb

```diff
@@ -32,9 +32,10 @@ def update
       end
 
       def order
-        params[:content_ids].each do |id|
-          content = Content.find(id)
-          content.move_to_bottom
+        Content.transaction do
+          params[:content_ids].each_with_index do |id, idx|
+            Content.where(id: id).update_all(position: idx + 1)
```

Comment:
I like this a lot, as it very obviously does what if's supposed to do. 

However, the `where` seems to indicate that you potentially update several records at the same time, which afaict is not the case. What about this:  
```
Content.find_by(id: id().update_column(:position, idx+1)
```